src/less/components/button.less

```diff
@@ -414,25 +422,13 @@
 
         &.btn-primary,
         &.btn-secondary,
-        &.btn-link {
-            &:after {
-                border-color: lighten(@brand-secondary, 40%);
-                border-left-color: @brand-secondary;
-            }
-        }
-
-        &.btn-executive-destructive {
-            &:after {
-                border-color: hsla(0, 0%, 100%, 0.5);
-                border-left-color: @brand-error;
-            }
-        }
-
+        &.btn-link,
+        &.btn-executive-destructive,
         &.btn-guiding-destructive,
         &.btn-link-destructive {
             &:after {
-                border-color: @brand-error-light;
-                border-left-color: @brand-error;
+                border-color: lighten(@brand-secondary, 40%);
+                border-left-color: @brand-secondary;
             }
         }
 
```

Comment:
Buttons with loading state should not be able to receive an active state
![image](https://user-images.githubusercontent.com/44967773/184603171-bcccadef-0a72-45ce-bfd8-70993f6f48b6.png)